app/mails/tools.py

```diff
@@ -24,9 +25,14 @@ def get_delay_days_from_email_address(email_address):
     :rtype: integer
     """
     try:
-        match = re.findall(r"^(\d+)([dmw])", email_address)[0]
-        multiplicator = settings.EMAIL_SUFFIX_TO_DAY[match[1]]
-        delay = int(match[0]) * int(multiplicator)
+        if re.match(r"^(\d+)([dmw])", email_address):
```

Comment:
I think you can get rid of that whole bit here. Dateparser can handle "5d", "4w", "3m" etc on it's own :-)

Also a few lines further down you overwrite the `delay` variable anyway with the result from the parser...

Reply:
I actually can't get rid of that part. Dateparser handles "3m" as 3 minutes and not months :roll_eyes:

Reply:
TBH I don't think it's worth the complexity. Most reminders will not be sent out for X months.

Instead of keeping the complexity, why not document that you'll need to write "3months@rmd.io" now instead of "3m" - and simplify the code? (Of course you'll need to update the addressbook generator as well)

Reply:
[Did you mean something like that?
](https://github.com/adfinis-sygroup/rmd.io/pull/94/commits/393c3458644c9fd9eb852e3f741d1956fa020429)